mutmut/__main__.py

```diff
@@ -533,6 +553,9 @@ def run_mutation(config, filename, mutation_id):
     finally:
         move(filename + '.bak', filename)
 
+        if config.post_mutation:
```

Comment:
we should add a simple integration test that ads some coverage over these statements just for good measure.

Maybe something like:

```python
def test_post_mutation_hook(filesystem):
    result = CliRunner().invoke(climain, ['run', '--paths-to-mutate=foo.py', "--test-time-base=15.0", """--post-mutation='python -c "print(\'post mutation stub\')""""], catch_exceptions=False)
    print(repr(result.output))
    assert result.exit_code == 0
    assert "post mutation stub" in result.output
```

Reply:
Good idea. I modified the test a little bit from what you wrote, and it did show a bug! I swallowed the output which would have been super annoying.